crates/core/component/dex/src/component/dex.rs

```diff
@@ -107,8 +107,11 @@ impl Component for Dex {
             .arbitrage(*STAKING_TOKEN_ASSET_ID, arb_routing_params)
             .await
         {
-            // The arb search completed, it may or may not have found surplus.
-            Ok(v) => tracing::info!(surplus = ?v, "arbitrage successful!"),
+            // The arb search completed successfully, and led to state changes.
+            Ok(Some(v)) => tracing::info!(surplus = ?v, "arbitrage successful!"),
+            // The arb completed without errors, but resulted in no surplus, so their
+            // state was not altered.
+            Ok(None) => tracing::info!("no arbitrage found"),
```

Comment:
Nit: I'd demote this to debug so that we only show the info if it actually happened, and omit it on an empty block.

Reply:
sounds good